### PR TITLE
test_format simplified

### PR DESCRIPTION
**Pull request by ywkim312**
test format file refactored to more simple way, so if there are new folder get created, you don't need to add the new path in test_format file anymore.

### CODE REVIEW

File: tests/test_format.py [hunk 1]
```diff
@@ -87,3 +21,5 @@ def test_conformance(paths=paths):
     style = pycodestyle.StyleGuide(quiet=False, max_line_length=120)
     result = style.check_files(paths)
     assert result.total_errors == 0
+
```
Comment (longshuicy) [inline tests/test_format.py:24]: you need to add one more blank line here

File: tests/test_format.py [hunk 2]
```diff
@@ -87,3 +21,5 @@ def test_conformance(paths=paths):
...
+test_conformance()
```
Comment (longshuicy) [inline tests/test_format.py:25]: you also need to add one blank line after this

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: tests/test_format.py (hunk 1 above)
Comment (ywkim312) [reply tests/test_format.py:24]: added

File: tests/test_format.py (hunk 2 above)
Comment (ywkim312) [reply tests/test_format.py:25]: added

**Review by longshuicy — APPROVED** [review]

**Comment by navarroc** [comment]
If I run the updated test_format, I see a LOT of pep8 errors, several hundred. Are you seeing the same? We should try to fix code as we update/modify it since we have so many pep8 issues.

**Comment by ywkim312** [comment]
> If I run the updated test_format, I see a LOT of pep8 errors, several hundred. Are you seeing the same? We should try to fix code as we update/modify it since we have so many pep8 issues.

Yes I see those. And I have created a issue for that https://github.com/IN-CORE/pyincore/issues/198

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+##[Unreleased]
+
+### Changed
+- Format test for the code simplified to included all the paths [#189](https://github.com/IN-CORE/pyincore/issues/189)
```
Comment (navarroc) [inline CHANGELOG.md:12]: This should be #193. Can you update the changelog?

Comment (ywkim312) [reply CHANGELOG.md]: modified... sorry

**Review by navarroc — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
+##[Unreleased]
+
+### Changed
+- Format test for the code simplified to included all the paths [#193](https://github.com/IN-CORE/pyincore/issues/193)
```
Comment (nunezco2) [inline CHANGELOG.md:12]: Fix typo:

"to include all"

Comment (ywkim312) [reply CHANGELOG.md]: fixed

**Pull request merged**